Sky Atlas/Sky Atlas.html

```diff
@@ -13221,25 +13230,26 @@ <h1>Sections & Primary Docs</h1>
           <td>Core</td>
           <td>In the near term pending development of the implementation of the model for Real World Assets, the following Instance Financial RRC Ratios will apply:
             <ul>
-              <li>BUIDL, JTRSY, and USTB - 0% Instance Financial RRC Ratio</li>
+              <li>BUIDL, JTRSY on Ethereum Mainnet, and USTB - 0% Instance Financial RRC Ratio</li>
+              <li>JTRSY on Avalanche - 0.5% Instance Financial RRC Ratio</li>
               <li>JAAA on Ethereum Mainnet - 1.6% Instance Financial RRC Ratio</li>
               <li>JAAA on Avalanche - 2.1% Instance Financial RRC Ratio</li>
             </ul>
-            In the near term, only BUIDL, JTRSY, USTB, and JAAA through Centrifuge v2 will be eligible for investment under this provision. All other investments in Real World Assets have a 100% Instance Total RRC as specified in <dfn>A.3.3 - Short Term Application To Real World Assets</dfn>.
-          </td>
+           <p> In the near term, only the assets specified in <dfn>A.3.3 - Short Term Application To Real World Assets</dfn> will be eligible for investment under this provision.</p><p>All other investments in Real World Assets have a 100% Instance Total RRC as specified in the document cited above.</p>
+         </td>
         </tr>
```

Comment:
a space has been incorrectly removed

```suggestion
          </td>
        </tr>
```